packages/app-form-builder/src/admin/components/FormEditor/Tabs/EditTab/EditTab.tsx

```diff
@@ -101,55 +98,80 @@ export const EditTab: React.FC = () => {
 
     const stepTitle = data.steps.find(step => step.id === isEditStep.id)?.title || "";
 
-    const handleStepMove = (source: any, step: FbFormStep): void => {
-        const { pos, formStep } = source;
-
-        if (pos) {
-            if (pos.index === null) {
+    const handleStepMove = (item: DragObjectWithFieldInfo, formStep: FbFormStep) => {
```

Comment:
Also let's try to use `useCallback` here. Maybe we can also create `useEditTab` hook here and make this file cleaner?